server/build.gradle

```diff
@@ -353,17 +365,8 @@ tasks.named("missingJavadoc").configure {
    */
   dependsOn("generateProto")
   javadocMissingIgnore = [
```

Comment:
Minor, we could drop `javadocMissingIgnore`, right?

Reply:
Are we not allowing for other exceptional cases?  I don't know what they would be, but if we remove this block entirely, when a need for such an exception rises in the future, this will need to be added back.  Or was this added just to handle this case that my PR completely solves?

Reply:
> Or was this added just to handle this case that my PR completely solves?

Yep, I believe it is solved by your pull request

Reply:
Done!  I do like removing code.  Less stuff to maintain!